src/dmd/dtoh.d

```diff
@@ -176,6 +178,33 @@ private bool isFrontendModule(ASTCodegen.Module m)
             (m.parent.parent.ident == DMDModule.dmd && !m.parent.parent.parent));
 }
 
+private string translateBasicType(ubyte ty)
+{
+    import AST = dmd.mtype;
+    switch (ty)
+    {
+        case AST.Tvoid:     return "void";
+        case AST.Tbool:     return "bool";
+        case AST.Tchar:     return "char";
+        case AST.Twchar:    return "char16_t";
+        case AST.Tdchar:    return "char32_t";
+        case AST.Tint8:     return "int8_t";
+        case AST.Tuns8:     return "uint8_t";
+        case AST.Tint16:    return "int16_t";
+        case AST.Tuns16:    return "uint16_t";
+        case AST.Tint32:    return "int32_t";
+        case AST.Tuns32:    return "uint32_t";
+        case AST.Tint64:    return "int64_t";
+        case AST.Tuns64:    return "uint64_t";
```

Comment:
This here is wrong, cppmangle doesn't map (u)long to (u)int64_t. It prefers C long if the size matches.

Reply:
... really?
Is that normal? And is that really the preferred behaviour? O_O